src/uct/api/v2/uct_v2.h

```diff
@@ -903,14 +922,14 @@ ucs_status_t uct_ep_query(uct_ep_h ep, uct_ep_attr_t *ep_attr);
  * check is a local operation it does not detect issues such as network
  * mis-configuration or lack of connectivity.
  *
- * @param [in]  iface       Local interface to check reachability from.
- * @param [in]  params      Operation parameters, see @ref
- *                          uct_iface_is_reachable_params_t.
+ * @param [in]    iface   Local interface to check reachability from.
+ * @param [inout] params  Operation parameters, see @ref
```

Comment:
why changed?

Reply:
@yosefe  `params` can't be const anymore -> the arg is `inout`.

Reply:
why should it be inout?

Reply:
Because `params` is not `const` anymore?

Reply:
unresolved

Reply:
```

    /**
     * User-provided pointer to a string buffer.
     * The function @ref uct_iface_is_reachable_v2 fills this buffer with a
     * null-terminated information string explaining why the remote address is
     * not reachable if the return value is 0.
     */
    char                          *info_string;
```